### DISCUSSION TITLE
How to use opencv on AvaloniaUI  and  pachages it as a deb for Linux installation

### DISCUSSION THREAD

**Discussion by d-moco** [AvaloniaUI/Avalonia#17269] [Q&A]
How to use opencv on AvaloniaUI and package it as a deb for Linux installation
OpenCVSHARP seems to support Windows

**Comment by timunie** [comment]
If they only support windows, you're tied to it I'm afraid.

**Comment by kekekeks** [comment]
You'll need to either compile and package it yourself, find a compatible binaries somewhere here - https://www.nuget.org/packages?q=opencvsharp.runtime&includeComputedFrameworks=true&prerel=true&sortby=relevance
or take https://www.nuget.org/packages/OpenCvSharp4_.runtime.ubuntu.20.04-x64 and supply it with extra libraries missing from your debian system via either patching or LD_PRELOAD
